apertium-get.py

```diff
@@ -0,0 +1,375 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+import re
+import sys
+import urllib.request
+from subprocess import (
+    run,
+    check_call,
+    check_output,
+    STDOUT,
+    DEVNULL,
+    CalledProcessError,
+)
+
+### Globals:
+
+pmodules = ["trunk", "staging", "nursery", "incubator"]
+lmodules = ["languages", "incubator"]
+
+git_ssh = "git@github.com:"
+git_https = "https://github.com/"
+
+apertium_git = "apertium/apertium-%s.git"
+giella_git = "giellalt/lang-%s.git"
+giella_core_git = "giellalt/giella-core.git"
+giella_shared_git = "giellalt/giella-shared.git"
+
+NOT_STARTED = 0
```

Comment:
Is this basically an enum? https://docs.python.org/3/library/enum.html

Reply:
yes

didn't know that existed - I'll switch it